Hannah-Sten/TeXiFy-IDEA
# LaTeX SDK

Fix #1589

#### Summary of additions and changes

https://jetbrains.org/intellij/sdk/docs/reference_guide/project_model/sdk.html#working-with-a-custom-sdk
- [x] Implement SDK
- [x] Implement custom step project generator for IntelliJ **problem**: SdkSettingsStep is only available in IntelliJ, so we cannot use it
~Implement custom step project generator for non-IntelliJ~
- [x] Check compatibility with non-IJ
- [x] Add option 'use project sdk' to run config dropdown for distribution
- [x] Clean up JdkComboBox and SdkSettingStep
- [x] Add MiKTeX SDK
- [x] Show a notification if pdflatex is not available, to hint to add to path/setup SDK

Also useful on mac?

#### How to test this pull request

* Make sure pdflatex is not in path, then setup sdk and try to run/use preview/reference installed packages
- [ ] (Need to wait before 2020.3 lands in flathub.....) Install IJ via flatpak, setup sdk and run


#### Wiki


- [x] Updated the wiki: https://github.com/Hannah-Sten/TeXiFy-IDEA/wiki/Latex-Sdk
